app/controllers/application_controller.rb

```diff
@@ -108,7 +108,9 @@ def corporate_categories
   def corporate_category?(category, corporate = corporate_categories, categories = [])
     categories << corporate.map {|c| c['id']}
     return true if categories.flatten.include?(category)
-    corporate_category?(category, corporate.first['contents'], categories.flatten) if corporate.first['contents']
+    unless corporate.first['contents']
```

Comment:
It would be, but right here corporate is just an `Array` that's put together from another method.